docker/cloudbuild.yaml

```diff
@@ -61,6 +61,17 @@ steps:
     docker push getsentry/sentry:$COMMIT_SHA
     docker tag us.gcr.io/$PROJECT_ID/sentry:$COMMIT_SHA getsentry/sentry:latest
     docker push getsentry/sentry:latest
+- name: 'node:12'
+  secretEnv: ['ZEUS_HOOK_BASE']
+  entrypoint: 'bash'
+  args:
+  - '-e'
+  - '-c'
+  - |
+    # Only push to Zeus from releases/*
+    # [[ ! "$BRANCH_NAME" =~ ^release/ ]] && exit 0
+    npx -p "@zeus-ci/cli" zeus job update --build $BUILD_ID --job 1 --ref $COMMIT_SHA --build-label "$(Build.BuildNumber)" --job-label "OSS Packages" --status passed
```

Comment:
You can just remove `--build-label`, or pass some build description if there's a title available in the environment. The job title looks fine, although I wonder if we can set a proper name so that it also shows up in GHA and use that?

Reply:
Note that `$(Build.BuildNumber)` is the human readable build name on Azure.